### ISSUE TITLE
Problems with limiting Post Types

### ISSUE THREAD

**Issue by imagefact** [deliciousbrains/wp-migrate-db-pro-bugs#16]
Perhaps I'm just not understanding proper usage, or this is a bug:

In my development site, I'm revising pages. But meanwhile every day my production site has new posts added. So I only want to bring pages over from development to production and leave posts alone.

Here's a list of what happens when I set Tables and Post Types in different ways with Migrate Pro 1.3 (not transferring anything else nor media):

------------------------------------ SETTINGS ------------------------------------------------------
Tables: Posts – Post Types: post
Brings in all posts but deletes all pages

Tables: Posts – Post Types: page
Brings in all pages but delete all posts

Tables: Postemeta & Posts  – Post Types:  post
Brings in all posts but deletes all pages

Tables: Postemeta & Posts – Post Types: page
Brings in all pages but delete all posts

Tables: postmeta – Post Types: page
Nothing happens

Tables: postmeta – Post Types: post
Nothing happens

---

Basically, it can bring over Posts or Pages, but it will erase the one it's not bringing over rather than leaving it alone.

Any solution for this?  Is it something I'm doing wrong? Or is this a bug?

Thanks!

**Comment by bradt** [comment]
Yes, this is the correct behavior. Tables are replaced when a migration is finalized, so that's why this happens.

**Comment by imagefact** [comment]
Huh?  Why would you want to ERASE tables you are not bringing over?

**Comment by bradt** [comment]
Post types are all kept in the same table: `wp_posts`.
